examples/platforms/efr32/sleepy-demo/sleepy-demo-ftd/CMakeLists.txt

```
@@ -27,6 +27,7 @@
 #
 
 add_executable(sleepy-demo-ftd
+    ${PROJECT_SOURCE_DIR}/openthread/examples/apps/cli/cli_uart.cpp
```

This causes a failure because we're in the openthread repo. You'll need to fix this for examples/platforms/efr32/sleepy-demo/sleepy-demo-mtd/CMakeLists.txt too

Fix below:

```suggestion
    ${PROJECT_SOURCE_DIR}/examples/apps/cli/cli_uart.cpp
```

---

Thanks @lmnotran ! Fixed.